src/main/groovy/life/qbic/portal/sampletracking/app/samples/query/SampleStatusOutput.groovy

```diff
@@ -0,0 +1,13 @@
+package life.qbic.portal.sampletracking.app.samples.query
+
+import life.qbic.datamodel.samples.Location
+
+interface SampleStatusOutput {
+
+    def availableLocations(List<Location> locations)
```

Comment:
I do not fully understand this class. It should implemented by the vaadin layout class. I do not understand the method names and where they are to be used in this case.

Maybe rename to `updateAvailableLocations`